cmd/config/config.go

```diff
@@ -292,9 +293,17 @@ using remote authorization you would do this:
 		if err != nil {
 			return err
 		}
-		return doConfig(args[0], in, func(opts config.UpdateRemoteOpt) (*fs.ConfigOut, error) {
+		err = doConfig(args[0], in, func(opts config.UpdateRemoteOpt) (*fs.ConfigOut, error) {
 			return config.CreateRemote(context.Background(), args[0], args[1], in, opts)
 		})
+		if err != nil {
+			return err
+		}
+
+		// Append the security warning about unencrypted configurations
```

Comment:
We should probably only show this message if the config isn't encrypted then we can make a definitive statement "your config is not encrypted..."

To do that we need to export the encrypted state from `fs/config/crypt.go` something like this (untested)

```go
func IsEncrypted() bool {
    return len(configKey) != 0
```

I think also that this should be printed to Stderr like the rest of the config stuff, and I'd probably hard wrap it to 80 chars.